core_backend/app/configs/app_config.py

```diff
@@ -29,3 +29,5 @@
 ACCESS_TOKEN_EXPIRE_MINUTES = os.environ.get("ACCESS_TOKEN_EXPIRE_MINUTES", 60 * 24 * 7)
 JWT_ALGORITHM = os.environ.get("JWT_ALGORITHM", "HS256")
 JWT_SECRET = os.environ.get("JWT_SECRET", "update-me-for-production")
+
+BACKEND_ROOT_PATH = os.environ.get("BACKEND_ROOT_PATH", "")
```

Comment:
This gets set in .env when using docker-compose to "/api". All this is needed to make sure that Swagger UI works when using nginx (at `/api/docs`) and when using local dev setup (at `/docs`)